components/script/dom/videotracklist.rs

```diff
@@ -75,40 +75,19 @@ impl VideoTrackList {
             .task_manager()
             .media_element_task_source_with_canceller();
 
+        track.set_selected(true);
```

Comment:
In the case where `value` is false, this does not behaves correctly.

Reply:
If that's the case, I wonder how was the old code working fine, as it would call that line both in the `if` and `else` blocks.

By the way, sorry for not having tested this yet. I'm still struggling to fix my servo environment (stuck on some error related to mozjs_sys) but I figured I could at least open the pull request as I assumed travis would check the changes.

Reply:
Yes, the od code also had the same incorrect behavior.